modules/EmailAddresses/EmailAddress.php

```diff
@@ -52,20 +52,56 @@
  */
 class EmailAddress extends SugarEmailAddress
 {
-	var $disable_row_level_security = true;
+    /**
+     * @var bool $disable_row_level_security
+     */
+	public $disable_row_level_security = true;
 
-	function __construct()
+    /**
+     * @var int|bool $opt_out
+     */
+    public $opt_out = 0;
+
+    /**
+     * @var int|bool $confirm_opt_in
+     */
+	public $confirm_opt_in = 0;
+
+	/**
+     * @var int|bool $invalid_email
+     */
+	public $invalid_email = 0;
+
+	/**
+     * @var TimeDate $opt_in_email_created
+     */
+    public $opt_in_email_created;
+
+    /**
+     * EmailAddress constructor.
+     */
+    function __construct()
```

Comment:
`function __construct()` visibility should be `public`

Reply:
done.